modules/cbQuestion/cbQuestion.php

```diff
@@ -849,5 +849,17 @@ public static function generateCSV($data, $delimiter = ',', $enclosure = '"') {
 		fclose($handle);
 		return $contents;
 	}
+
+	public static function getQnDelimeterProperty($qnid) {
+		$ans = self::getAnswer($qnid);
```

Comment:
This is very expensive in resources just to get the properties. We calculate the whole question's answer just to get the properties. Can we create another new static method in cbQuestion `getQuestionProperties`  that returns just the properties and then use that one in this method?

Reply:
Sure, 'll update it.